scripts/robo-collector-commit

```diff
@@ -32,7 +32,7 @@ clone_collector() {
     # Redirect to devnull so this doesn't print the value of the token on
     # failure.
     git clone\
-        https://roxbot:$GITHUB_TOKEN@github.com/stackrox/collector.git\
+        https://roxbot:$ROBOT_ROX_GITHUB_TOKEN@github.com/stackrox/collector.git\
```

Comment:
How does this not breaking crawling with OSCI?

Reply:
Excellent question. I'll make sure this is compatible